src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,8 +36,8 @@ public static boolean isValidAddress(String test) {
     }
 
     @Override
-    public String toString() {
-        return value;
```

Comment:
If you are removing `toString`, remember to remove it in the super class as well, and to update any relevant documents, comments, or tests.